crates/glaredb/src/local.rs

```diff
@@ -158,22 +75,29 @@ impl LocalSession {
         .await?;
 
         let sess = if let Some(url) = opts.cloud_url.clone() {
-            let exec_client = if opts.ignore_rpc_auth {
-                RemoteClient::connect(url).await?
+            let (exec_client, info_msg) = if opts.ignore_rpc_auth {
+                let u = &url.to_string();
+                (
+                    RemoteClient::connect(url).await?,
+                    format!("Connected to remote GlareDB server: {}", u.bright_cyan()),
```

Comment:
Very nitpicky, bright cyan is hard to read on light themes (blasphemy I know). Could we make this and the one below just cyan?